package.json

```diff
@@ -10,9 +10,10 @@
   ],
   "author": "WalletConnect <walletconnect.org>",
   "homepage": "/testwallet",
+  "homepage-broken": ".",
```

Comment:
what's this?

Reply:
leftovers from experiments.  I'll remove.